### PR TITLE
Uno fix

### PR DESCRIPTION
**Pull request by Gusabary**

### CODE REVIEW

File: core/provider.proto [hunk 1]
```diff
@@ -37,7 +37,7 @@ message RegisterArgs {
 }
 
 message RegisterRet {
-  ErrorNumber errno = 1;
```
Comment (Veiasai) [inline core/provider.proto:40]: ??? give me a reason.

File: uno/uno.proto [hunk 2]
```diff
@@ -12,26 +12,121 @@ message StartGameSettings {
...
+
+message DoubtRsp {
+    int32 doubter = 1;
+    int32 doubted = 2;
+    bool success = 3;
+    // the player doubted will show his handcards
+    repeated Card handcards = 4;
+}
+
+message JoinGame {
```
Comment (Veiasai) [inline uno/uno.proto]: you don't need JoinGame

File: core/provider.proto (hunk 1 above)
Comment (Gusabary) [reply core/provider.proto:40]: `errno` will be generated literally by `protoc`, which is a macro in cpp however

File: uno/uno.proto (hunk 2 above)
Comment (Gusabary) [reply uno/uno.proto]: yep, forgot to remove that. Already fixed now

**Review by Veiasai — APPROVED** [review]

**Review by Gusabary — COMMENTED** [review]
cool

**Pull request merged**